src/storage.rs

```diff
@@ -96,6 +96,9 @@ where
     /// Returns true if the log contains a StopSign or a StopSign already has been decided.
     /// Note that the log could have a StopSign that later gets overwritten, and thus this function might first return true and later false.
     fn stopped(&self) -> bool;
+
+    /// Starts the garbage collection process to free storage.
```

Comment:
This function does not start the garbage collection process (which is done in the `OmniPaxos` struct), it is rather just removing the first `idx` elements of the log.

```suggestion
    /// Removes elements up to the given [`idx`] from storage.
```